py/server/deephaven/table_listener.py

```diff
@@ -23,6 +23,7 @@
 _JListenerRecorder = jpy.get_type("io.deephaven.engine.table.impl.ListenerRecorder")
 _JPythonMergedListenerAdapter = jpy.get_type("io.deephaven.integrations.python.PythonMergedListenerAdapter")
 
+_DEFAULT_ON_ERROR_CALLBACK = lambda e : print(f"An error occurred during listener execution: {e}")
```

Comment:
Is there a reason the default callaback isn't being called in the default methods for the 2 listener classes?

Reply:
The default methods include the listener object (its repr) in the message as well.